python/GangaLHCb/test/GPI/TestDataFiles.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+from Ganga.testlib import external
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+
+class TestDataFiles(GangaUnitTest):
+
+    @external
+    def testDataFiles(self):
+
+        from Ganga.GPI import DiracFile, config
+
+        # LFNs
+        name = 'test.txt'
+        lfn = DiracFile(lfn=name)
+        assert lfn.lfn == name
+        lfn = DiracFile('lfn:'+name)
+        assert lfn.lfn == name
+        lfn = DiracFile('LFN:'+name)
+        assert lfn.lfn == name
+        try:
+            lfn = DiracFile('pfn:'+name)
+            raise RuntimeError('Should have got exception trying to create DiracFile from pfn.')
+        except:
+            pass
```

Comment:
Actually, this doesn't work in any case as the `raise`'d exception just gets caught anyway :smile: Plus it **doesn't** raise an exception. So I've just remove the `try...except` block.